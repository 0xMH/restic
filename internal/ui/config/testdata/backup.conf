backup {
    target = [
        "foo",
        "/home/user",
    ]
}
