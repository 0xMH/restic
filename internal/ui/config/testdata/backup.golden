{
  "Backends": null,
  "Repo": "",
  "Backup": {
    "Target": [
      "foo",
      "/home/user"
    ]
  }
}
